product_docs/docs/pgd/5/rel_notes/index.mdx

```diff
@@ -2,7 +2,9 @@
 title: "EDB Postgres Distributed Release notes"
 navTitle: "Release notes"
 navigation:
+- pgd_5.0.1_rel_notes
```

Comment:
Yes, I don't think that was ever possible or practical. Anyway, I think this is just a doc issue now and can be resolved by restructuring the pages a little bit but yes it will need discussion with wider audience (in fact, it looks ok for this release but this will pop up again if there are new releases).

Reply:
At minimum, I can remove the TPA column. Anything else you'd like we to do?

Reply:
Yes, that should be fine for now I guess.